scripts/set-image-version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eu
+
+source vars
+
+#if distribution is 2.x download latest image, if images is 1.x download 1.x image
+# setups container name
+if [ "$Version" == "1.1.0-SNAPSHOT" ];
+then
```

Comment:
Don't hard-code 1.1.0-SNAPSHOT, that could theoretically change.  This version will always start with "1." though.